gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemXAttrsIntegrationTest.java

```diff
@@ -33,38 +33,40 @@
 import java.util.EnumSet;
 import java.util.HashMap;
 import java.util.Map;
-import org.apache.hadoop.fs.FileSystem;
 import org.apache.hadoop.fs.Path;
 import org.apache.hadoop.fs.XAttrSetFlag;
-import org.junit.AfterClass;
-import org.junit.BeforeClass;
+import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
 @RunWith(JUnit4.class)
-public final class GoogleHadoopFileSystemXAttrsIntegrationTest {
+public final class GoogleHadoopFileSystemXAttrsIntegrationTest extends HadoopFileSystemTestBase {
```

Comment:
This seems out of scope of this PR? Also, I do not think that we need to follow inheritance pattern in tests that do not use it.